core/foundation/inc/ROOT/RLogger.hxx

```diff
@@ -342,7 +342,7 @@ inline ELogLevel RLogChannel::GetEffectiveVerbosity(const RLogManager &mgr) cons
  to prevent "ambiguous else" in invocations such as `if (something) R__LOG_DEBUG()...`.
  */
 #define R__LOG_TO_CHANNEL(SEVERITY, CHANNEL)                                                                        \
-   (SEVERITY < ROOT::Experimental::ELogLevel::kInfo ||                                                              \
+   (SEVERITY < ROOT::Experimental::ELogLevel::kInfo + 0 ||                                                          \
```

Comment:
Can you add parenthesis to clarify?  I.e something like:
```suggestion
   ( ( (SEVERITY < ROOT::Experimental::ELogLevel::kInfo) + 0 ) ||                                                          \
```
but I don't know if it suppresses  the warning

Reply:
I do not find that an improvement in clarity. I find `(a < b + 0) || ` a reasonable compromise ;-) so that's what I'll do.

Reply:
@pcanal better now?